### ISSUE TITLE
Localize our existing emails

### ISSUE THREAD

**Issue by lucca65** [cambiatus/backend#197]
# Story

I as a community member
Want to receive emails with my preferred language
To actually understand what's written there

### Telas

N/A

### Use cases

Given a new user joined the community 
When the system is sending the welcome email
Then its on the preferred language

Given a new claim was approved
When the system is sending the approved claim email
Then its on the preferred language

Given a new transfer happened
When the system is sending the transfer email
Then its on the preferred language

### Description

Although we have emails, we always send it in English instead of being localized like the frontend

**Comment by lucca65** [comment]
Depends on #196

**Comment by lucca65** [comment]
Already done on #200
